backend/src/StamAcasa.JobScheduler/Program.cs

```diff
@@ -44,18 +48,38 @@ private static void Main(string[] args)
                         userName,
                         password,
                         10, //default
-                        (x) => {}));
+                        (x) => { }));
 
                     services.TryAddScheduledJob<HealthCheckJob>();
                     services.TryAddScheduledJob<SendAssessmentReminderJob>(s =>
                     {
                         s.TryAddTransient<AssessmentNotificationsDispatch>();
                         s.TryAddSingleton<IQueueService, QueueService>();
                         s.TryAddTransient<IUserService, UserService>();
+
                     });
+
                     services.TryAddScheduledJob<SendResultsNotificationJob>(s =>
                     {
                         s.TryAddTransient<ResultNotificationsDispatch>();
+                        s.TryAddTransient<IFormService, FormService>();
+                        s.TryAddTransient<IAnswersExcelExporter, AnswersExcelExporter>();
+                        s.TryAddTransient<IExcelDocumentService, ExcelDocumentService>();
+                        s.AddSingleton<IEmailSender>(ctx =>
+                            new SmtpSender(
+                                new SmtpOptions
```

Comment:
But should we be using the IEmailSender directly or the `IQueueService` ? Like we do  [here](https://github.com/code4romania/stam-acasa/blob/4e57bbbecc372776c3105d1db8642776473e3985/backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs#L90)

Reply:
You're right, I didn't know about the IQueueService, so I thought we will either use the SendGrid sender or the SMTP implementation. I'll push another commit and replace it with IQueueService.

Reply:
Yup, please switch to the IQueueService here. Thank you!